HLSPlugin/src/com/kaltura/hls/m2ts/TSPacketParser.as

```diff
@@ -214,7 +217,7 @@ package com.kaltura.hls.m2ts
             }
 
             // Append to buffer.
-            if(!pesProcessor.append(new PESPacket(stream.packetID, stream.buffer)))
+            if(!pesProcessor.append(new PESPacket(stream.packetID, stream.buffer),pesProcessor.transcoder.id3Callback))
```

Comment:
Given the pesProcessor has the transcoder, is the callback needed as an argument?